openlibrary/core/lists/engine.py

```diff
@@ -4,7 +4,6 @@
 import collections
 import re
 
-import simplejson
```

Comment:
simplejson is not in the Python standard library.

Reply:
@cclauss Added simplejson